test/micromamba/test_linking.py

```diff
@@ -100,9 +100,9 @@ def test_cross_device(self, allow_softlinks, always_copy):
 
         create_args = ["xtensor", "-p", "/tmp/testenv", "--json"]
         if allow_softlinks:
-            create_args.append("--allow-softlinks", no_dry_run=True)
+            create_args.append("--allow-softlinks")
         if always_copy:
-            create_args.append("--always-copy", no_dry_run=True)
```

Comment:
I think we should keep the `no_dry_run=True` here because we check if we're linking the packages correctly by looking at inode etc.

Reply:
Uh, that fails when I run the test, saying you can't have keyword args in `list.append`? `no_dry_run=True` is there just below in the call to `create`.